yield_plugin/src/yield_plugin.cpp

```diff
@@ -726,7 +785,7 @@ namespace yield_plugin
     return result;
   }
 
-  double YieldPlugin::max_trajectory_speed(const std::vector<carma_planning_msgs::msg::TrajectoryPlanPoint>& trajectory_points) const
+  double YieldPlugin::max_trajectory_speed(const std::vector<carma_planning_msgs::msg::TrajectoryPlanPoint>& trajectory_points, double timestamp_to_search_until) const
```

Comment:
I renamed it to `timestamp_in_sec_to_search_until` instead so that we don't have to make a lot of conversions from rclcpp::Time to double and back and forth